wcivf/apps/elections/templates/elections/includes/_single_ballot.html

```diff
@@ -56,7 +56,7 @@ <h2>
                                 </span>
                             </h2>
                         </summary>
-                        <p>{% blocktrans with description=postelection.election.description|markdown %} {{ description }}{% endblocktrans%}</p>
+                        <div>{% blocktrans with description=postelection.election.description|markdown %} {{ description }}{% endblocktrans%}</div>
```

Comment:
nested <p> is invalid and `|markdown` will generate HTML containing `<p></p>`s

Reply:
Sorry, what do you mean by "nested p is invalid"? This is a `p` in a `details`, what's wrong with that?

If the `|markdown` is going to make a `p`, can we just remove the wrapping `div`?

Reply:
There's nothing wrong with a `<p>` in a `<details>`

`<p>` inside another `<p>` is invalid.

`|markdown` will generate at least one `<p>`. I've just removed the container element (`<div>`) completely.

Reply:
Got it, sorry I got confused by that :+1: